### ISSUE TITLE
It would be a good idea to document what would be the increased risk of failure resulting from selecting a subset of available blockdevs to write metadata to

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/stratis-docs#45]

**Comment by agrover** [comment]
How do you mean? If we have a 1000 disk pool for example and write metadata to 10, if those 10 somehow end up not being in the pool after a reboot, then we will get stale metadata and that could be extremely bad.

**Comment by mulkieran** [comment]
Yes. That's one failure mode. But we should understand it in terms of probabilities, like people are able to do with RAID.
